gnosis/eth/account_abstraction/user_operation_receipt.py

```diff
@@ -44,6 +45,17 @@ def from_bundler_response(
             user_operation_receipt_response["logs"],
         )
 
+    def get_deployed_account(self) -> Optional[ChecksumAddress]:
+        """
+        :return: Deployed account in case a new account was deployed
+        """
+        for log in self.logs:
+            if (
+                len(log["topics"]) == 3
+                and HexBytes(log["topics"][0]) == ACCOUNT_DEPLOYED_TOPIC
+            ):
+                return fast_to_checksum_address(log["topics"][2][-40:])
```

Comment:
Maybe we could add some link documentation why `[2][-40:]`

Reply:
Done, thanks!